helpers/foodBooking.helper.js

```diff
@@ -83,35 +102,61 @@ export async function bookFoodForMumukshus(
     );
   }
 
-  const allDates = getDates(start_date, end_date);
+  const utsav =
+    primary_booking.booking_type == TYPE_UTSAV
+      ? await UtsavDb.findOne({
+          where: { id: primary_booking.details.utsavid }
+        })
+      : null;
+
+  const allDates = getDatesDuringUtsav(start_date, end_date, utsav);
```

Comment:
@vendz - this was only happening previously for guests, but now this logic will also apply to food bookings for mumukshus and self. I think thats desirable. lmk.